index.html

```diff
@@ -15,7 +15,40 @@
             <nav></nav>
         </header>
         <main></main>
-        <footer></footer>
+        <footer>
+            <div class="container">
+                <nav class="footer-nav">
+                    <ul>
+                        <li>
+                            <a href="hospitals.html" title="Spitale">Spitale</a> 
+                        </li>
+                        <li>
+                            <a href="deparments.html" title="Sectii">Sectii</a> 
+                        </li>
+                        <li>
+                            <a href="doctors.html" title="Doctori">Doctori</a> 
+                        </li>
+                        <li>
+                            <a href="contact.html" title="Contact">Contact</a> 
+                        </li>
+                    </ul>    
+                </nav>
+                <div class="social-media">
+                    <a href="https://www.facebook.com" title="Go to Facebook page" target="_blank">
+                        <img src="assets\img\facebook.png" alt="Facebook link" width="25" height="25"></a>
```

Comment:
Width and height attributes are kindof deprecated. Try using a CSS class instead.